app/controllers/api/projects.php

```diff
@@ -130,18 +140,50 @@
             }
         }
 
-        $databaseOverride = System::getEnv('_APP_DATABASE_OVERRIDE', null);
-        $index = array_search($databaseOverride, $databases);
+        $databaseOverride = App::getEnv('_APP_DATABASE_OVERRIDE');
+        $index = \array_search($databaseOverride, $databases);
         if ($index !== false) {
-            $database = $databases[$index];
+            $dsn = $databases[$index];
         } else {
-            $database = $databases[array_rand($databases)];
+            $dsn = $databases[array_rand($databases)];
         }
 
         if ($projectId === 'console') {
             throw new Exception(Exception::PROJECT_RESERVED_PROJECT, "'console' is a reserved project.");
         }
 
+        // TODO: One in 20 projects use shared tables. Temporary until all projects are using shared tables.
+        if (
+            !\mt_rand(0, 19)
+            && App::getEnv('_APP_DATABASE_SHARED_TABLES', 'enabled') === 'enabled'
+            && App::getEnv('_APP_EDITION', 'self-hosted') !== 'self-hosted'
+        ) {
+            $schema = 'appwrite';
```

Comment:
why is the schema `appwrite` ? 

Firstly shouldn't it be `scheme` ? and shouldn't the scheme be `mysql` ?

Reply:
The intention was to use it as the DB schema so we could support it for other engines that allow schema, mainly Postgres. In that case, hardcoding mysql wouldn't work well